simulariumio/smoldyn/smoldyn_data.py

```diff
@@ -71,3 +80,152 @@ def __init__(
             spatial_units if spatial_units is not None else UnitData("m")
         )
         self.plots = plots if plots is not None else []
+
+    @classmethod
+    def from_buffer_data(
```

Comment:
Change name to be something more like `from_dict`

Reply:
and also name the other `from_buffer_data` methods to `from_dict` if they arent actually unpacking a buffer